xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml

```diff
@@ -103,12 +103,12 @@
       </Docs>
     </Member>
     <Member MemberName="CreateArrayInfo&lt;TElement&gt;">
-      <MemberSignature Language="C#" Value="public static System.Text.Json.Serialization.Metadata.JsonTypeInfo&lt;TElement[]&gt; CreateArrayInfo&lt;TElement&gt; (System.Text.Json.JsonSerializerOptions options, System.Text.Json.Serialization.Metadata.JsonTypeInfo elementInfo, System.Text.Json.Serialization.JsonNumberHandling numberHandling, Action&lt;System.Text.Json.Utf8JsonWriter,TElement[]&gt;? serializeFunc);" />
-      <MemberSignature Language="ILAsm" Value=".method public static hidebysig class System.Text.Json.Serialization.Metadata.JsonTypeInfo`1&lt;!!TElement[]&gt; CreateArrayInfo&lt;TElement&gt;(class System.Text.Json.JsonSerializerOptions options, class System.Text.Json.Serialization.Metadata.JsonTypeInfo elementInfo, valuetype System.Text.Json.Serialization.JsonNumberHandling numberHandling, class System.Action`2&lt;class System.Text.Json.Utf8JsonWriter, !!TElement[]&gt; serializeFunc) cil managed" />
-      <MemberSignature Language="DocId" Value="M:System.Text.Json.Serialization.Metadata.JsonMetadataServices.CreateArrayInfo``1(System.Text.Json.JsonSerializerOptions,System.Text.Json.Serialization.Metadata.JsonTypeInfo,System.Text.Json.Serialization.JsonNumberHandling,System.Action{System.Text.Json.Utf8JsonWriter,``0[]})" />
-      <MemberSignature Language="VB.NET" Value="Public Shared Function CreateArrayInfo(Of TElement) (options As JsonSerializerOptions, elementInfo As JsonTypeInfo, numberHandling As JsonNumberHandling, serializeFunc As Action(Of Utf8JsonWriter, TElement())) As JsonTypeInfo(Of TElement())" />
-      <MemberSignature Language="F#" Value="static member CreateArrayInfo : System.Text.Json.JsonSerializerOptions * System.Text.Json.Serialization.Metadata.JsonTypeInfo * System.Text.Json.Serialization.JsonNumberHandling * Action&lt;System.Text.Json.Utf8JsonWriter, 'Element[]&gt; -&gt; System.Text.Json.Serialization.Metadata.JsonTypeInfo&lt;'Element[]&gt;" Usage="System.Text.Json.Serialization.Metadata.JsonMetadataServices.CreateArrayInfo (options, elementInfo, numberHandling, serializeFunc)" />
-      <MemberSignature Language="C++ CLI" Value="public:&#xA;generic &lt;typename TElement&gt;&#xA; static System::Text::Json::Serialization::Metadata::JsonTypeInfo&lt;cli::array &lt;TElement&gt; ^&gt; ^ CreateArrayInfo(System::Text::Json::JsonSerializerOptions ^ options, System::Text::Json::Serialization::Metadata::JsonTypeInfo ^ elementInfo, System::Text::Json::Serialization::JsonNumberHandling numberHandling, Action&lt;System::Text::Json::Utf8JsonWriter ^, cli::array &lt;TElement&gt; ^&gt; ^ serializeFunc);" />
+      <MemberSignature Language="C#" Value="public static System.Text.Json.Serialization.Metadata.JsonTypeInfo&lt;TElement[]&gt; CreateArrayInfo&lt;TElement&gt; (System.Text.Json.JsonSerializerOptions options, System.Text.Json.Serialization.Metadata.JsonCollectionInfoValues&lt;TElement[]&gt; collectionInfo);" />
```

Comment:
Should we filter JsonMetadataServices type altogether?

Reply:
@layomia Should we filter JsonMetadataServices and JsonCollectionInfoValues<T> from appearing on docs?

Reply:
@eerhardt Should we filter JsonMetadataServices and JsonCollectionInfoValues from appearing on docs? I'm guessing we can't filter the whole System.Text.Json assembly, so I can filter at the type level if necessary.

Reply:
The types are marked EditorBrowsableState.Never with a remarks section that you shouldn't call them directly:

https://github.com/dotnet/runtime/blob/0681dd60aca47a1efd41d883c42c4d2afa47168c/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonMetadataServices.cs#L11-L13

They are real public APIs, in an assembly that users reference. So I don't think they should be filtered out. If we filtered them out, then no one will be able to see those remarks.